lib/frontend/admin/pages/article/article_add.dart

```diff
@@ -166,27 +169,38 @@ class _AddBlogPostState extends State<AddBlogPost> {
               isEnabled: isEnabled,
               tags: tagsController.text.split(','),
               author: authorController.text,
-              content: await controller.getText(),
+              content: await quillEditorcontroller.getText(),
```

Comment:
There is no point of adding a try and catch for this controller, there is already whole try and catching, making sure if there is an error then it would catch it and then display it in the UI as well,